client/src/Component/videos/videoForm.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import {backEndApi } from "../../config/config.js";
+
+const VideoForm = ({ handleAddVideo }) => {
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    console.log("Form submitted");
+    if (title.trim() === "" || !isValidUrl(url)) {
+      return;
+    }
+
+    try {
+      const response = await fetch(`${backEndApi}`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ title, url }),
+      });
+
+      if (response.ok) {
+        setTitle("");
+        setUrl("");
+      } else {
+        console.error("Error adding video:", response.statusText);
```

Comment:
This text doesn't show up anywhere the user will see - how could we better make it so that the user will know what went wrong and what they should do about it?